main.py

```diff
@@ -0,0 +1,92 @@
+import os
+import requests
+import boto3
+import logging
+from dotenv import load_dotenv
+from json_log_formatter import JsonFormatter
+
+def setup_logger():
+    # Create a custom logger
+    logger = logging.getLogger(__name__)
+
+    # Create handler
+    handler = logging.StreamHandler()
+
+    # Create formatter
+    formatter = JsonFormatter()
+
+    # Add formatter to handler
+    handler.setFormatter(formatter)
+
+    # Add handler to logger
+    logger.addHandler(handler)
+
+    # Set level of logging
+    logger.setLevel(logging.INFO)
+    return logger
+
+def get_env_vars():
+    ORGANIZATION = os.getenv('ORGANIZATION')
+    S3_BUCKET = os.getenv('S3_BUCKET')
+    TOKEN = os.getenv('TOKEN')
+
+    env_vars = {'ORGANIZATION': ORGANIZATION, 'S3_BUCKET': S3_BUCKET, 'TOKEN': TOKEN}
+    missing_vars = [var for var, value in env_vars.items() if not value]
+
+    if missing_vars:
+        for var in missing_vars:
+            logger.error(f'Environment variable {var} is not set.')
+        exit(1)
+
+    return ORGANIZATION, S3_BUCKET, TOKEN
+
+def get_workspaces(ORGANIZATION, TOKEN):
+    headers = {
+        'Authorization': f'Bearer {TOKEN}',
+        'Content-Type': 'application/vnd.api+json'
+    }
+
+    url = f'https://app.terraform.io/api/v2/organizations/{ORGANIZATION}/workspaces'
+    response = requests.get(url, headers=headers)
+
+    if response.status_code != 200:
```

Comment:
I'm not sure what types of responses come back from TFC, but there are other codes that can indicate a successful response, e.g.  201 (Created), 202 (Accepted).

you can look at using the response.ok boolean to accommodate these, since you could get an error on a 201 or 202.